aptos-move/aptos-e2e-comparison-testing/src/execution.rs

```diff
@@ -419,11 +415,8 @@ impl Execution {
 
         // Update features if needed to the correct binary format used by V2 compiler.
         let mut features = Features::fetch_config(&state).unwrap_or_default();
-        self.enable_language_related_features(&mut features);
-        if self.disable_aa_fa {
-            self.disable_aa(&mut features);
-            self.disable_fa_store(&mut features);
-        }
+        self.enable_features(&mut features, &self.enable_features);
```

Comment:
So now all language features and AA handling are done by the caller, right? That's nice!